gulp/tasks/default.js

```diff
@@ -1,5 +1,10 @@
-(function() {
-    'use strict';
-    var gulp = require('gulp');
-    gulp.task('default', ['test', 'lint', 'jscs', 'coverage']);
-}());
+'use strict';
+
+var gulp = require('gulp');
+
+gulp.task('default', [
+    'doc-build',
+    'lint',
+    'jscs',
```

Comment:
Note that we're going to switch over to ESLint which will then remove the need for JSCS entirely:

https://openedx.atlassian.net/browse/FEDX-164